src/styles/layout/_page.scss

```diff
@@ -0,0 +1,34 @@
+$tablet-size: 600px;
+$wide-screen-size: 1000px;
```

Comment:
better create separate file for variables

Reply:
Did that